bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -268,6 +269,13 @@ private constructor(
     @get:JvmName("suggestions")
     val suggestions: SuggestionMethods by lazy { SuggestionMethods(this) }
 
+    /**
+     * Access API methods under "api/vX/push" endpoint.
+     */
+    @Suppress("unused") // public API
+    @get:JvmName("pushNotifications")
+    val pushNotification: PushNotificationMethods by lazy { PushNotificationMethods(this) }
```

Comment:
```suggestion
    val pushNotifications: PushNotificationMethods by lazy { PushNotificationMethods(this) }
```

We should rename this to match both the `JvmName` and the naming of other methods in this client